libsolidity/analysis/TypeChecker.cpp

```diff
@@ -199,7 +196,7 @@ void TypeChecker::checkContractAbstractFunctions(ContractDefinition const& _cont
 				if (!function->isImplemented())
 					// Base contract's constructor is not fully implemented, no way to get
 					// out of this.
-					allBaseConstructorsImplemented = false;
+					_contract.annotation().unimplementedFunctions.push_back(function);
```

Comment:
I think this case is different from the other "unimplemented functions" case. For error reporting, it can be the same, but I think the annotations deserve a separate boolean for this case. Please also document that `unimplementedFuctions` can also contains functions from base classes, especially constructors.